en/developer_guide/external_media_domain.rst

```diff
@@ -0,0 +1,25 @@
+Adding an external media domain
+===============================
+
+Configuration
+~~~~~~~~~~~~~
+
+Sometimes, developers want an external media domain to improve websites performances.
+Open Orchestra gives you an easy way to add an external media domain in the configuration:
```

Comment:
Just a general remark : this expression ("Open Orchestra gives you a way to do smth") is widely used in the docs. I personally think it's a heavy way of explaining things.